### ISSUE TITLE
`parameters.add_exchanges_to_group` - can't add multiple activities with only Database parameters

### ISSUE THREAD

**Issue by aleksandra-kim** [brightway-lca/brightway2-data#56]
**[Original report](https://bitbucket.org/cmutel/brightway2-data/issue/56) by James Joyce (Bitbucket: [pjamesjoyce](https://bitbucket.org/pjamesjoyce), GitHub: [pjamesjoyce](https://github.com/pjamesjoyce)).**

----------------------------------------

Adding multiple activities with formulas containing only database parameters to a group (e.g. "all") causes an error because the dummy parameters created at activity level are not unique. 

For example:
```python
for activity in database:
    for exchange in activity.exchanges():
        if exchange.get('formula'):
            bw2.parameters.add_exchanges_to_group("all", activity)
            break
```
Because there are no activity level parameters specified, `parameters.add_exchanges_to_group` adds one called "\_\_dummy\_\_", then adds the exchanges to the `ParameterizedExchange` table.

This works first time around, but then fails for the second activity because "\_\_dummy\_\_" already exists in the group "all".

**Comment by aleksandra-kim** [comment]
**Original comment by Chris Mutel (Bitbucket: [cmutel](https://bitbucket.org/cmutel), GitHub: [cmutel](https://github.com/cmutel)).**

----------------------------------------

3.4.2. Fixes #56

**Comment by aleksandra-kim** [comment]
**Original comment by Chris Mutel (Bitbucket: [cmutel](https://bitbucket.org/cmutel), GitHub: [cmutel](https://github.com/cmutel)).**

----------------------------------------

3.4.2 released on pypi and conda.
